tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -125,79 +125,79 @@
 // Delegate for testing Cortana foreground activation
 // Use text value to guarantee it is the same argument we create
 @interface CortanaVoiceCommandForegroundTestDelegate : NSObject <UIApplicationDelegate>
-@property (nonatomic, readonly) NSMutableDictionary* methodsCalled;
+@property (nonatomic, readonly) StrongId<NSMutableDictionary> methodsCalled;
```

Comment:
Since this is test code, I dont have a strong preference.  In framework code, we have been doing 1 consistently, we should probably do the same here.  (adding -dealloc is quite simple and less intrusive).

---
In reply to: [82303181](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82303181) [](ancestors = 82303181,82302619,82297024,82296469)